src/main/java/com/example/Ingress_lab/Application.java

```diff
@@ -1,13 +1,31 @@
 package com.example.Ingress_lab;
 
+import com.example.Ingress_lab.service.CustomerService;
+import com.example.Ingress_lab.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+@RequiredArgsConstructor
 @SpringBootApplication
-public class Application {
+public class Application implements CommandLineRunner {
+	private final UserService userService;
+	private final CustomerService customerService;
+
 	public static void main(String[] args) {
 		SpringApplication.run(Application.class, args);
 
 	}
 
+
+	@Override
+	public void run(String... args) throws Exception {
```

Comment:
[Optional]: not implement
@Bean
CommandLineRunner commandLineRunner() {
  return args -> {
      // implement code
  }
}